MUXSDKStats/MUXSDKStats/MUXSDKPlayerBinding.m

```diff
@@ -668,11 +674,28 @@ - (void)dispatchTimeUpdateEvent:(CMTime)time {
 }
 
 - (void)dispatchError {
+    if (!_automaticErrorTracking) {
+        return;
+    }
+    if (![self isPlayerOK]) {
+        return;
+    }
+    [self checkVideoData];
+    MUXSDKPlayerData *playerData = [self getPlayerData];
+    MUXSDKErrorEvent *event = [[MUXSDKErrorEvent alloc] init];
+    [event setPlayerData:playerData];
+    [MUXSDKCore dispatchEvent:event forPlayer:_name];
+    _state = MUXSDKPlayerStateError;
+}
+
+- (void)forceDispatchError:(nonnull NSString *)code withMessage:(nonnull NSString *)message {
```

Comment:
maybe a better name here?

Reply:
Usually you'd have a method that allows you to force or not force using a boolean 🤔 
`(void)dispatchError:(nonnull NSString *)code withMessage:(nonnull NSString *)message forPlayer:(nonnull NSString *)name force:(BOOL) force;`